jax/_src/test_util.py

```diff
@@ -504,8 +504,9 @@ def device_supports_buffer_donation():
   )
 
 
+@contextmanager
 def set_host_platform_device_count(nr_devices: int):
-  """Returns a closure that undoes the operation."""
+  """Context manager to set host platform device count if not specified by user."""
```

Comment:
Can we say that you cannot decorate methods with this? This is only supposed to be applied at the top level in setUpModule?

Reply:
Done